examples/stm32h7-example/Core/Inc/cli_binding.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * cli_binding.h
+ *
+ *  Created on: Jul 7, 2023
+ *      Author: NeusAap
+ */
+
+#ifndef INC_CLI_BINDING_H_
+#define INC_CLI_BINDING_H_
+
+void initCliBinding();
```

Comment:
Please add some comments here what these functions do since they are "public". It will be faster to see it in code than to open README

Reply:
Whoops, added the comments to the source file. Forgot to add it to the header. Good catch.